CHANGELOG.md

```diff
@@ -141,6 +141,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 
+- Fixed `fuse_modules` to be qat-aware for `pytorch>1.10`. ([#12891](https://github.com/PyTorchLightning/pytorch-lightning/pull/12891))
```

Comment:
The change here impacts 1.10 as well, right?
```suggestion
- Fixed `fuse_modules` to be qat-aware for `torch>=1.10`. ([#12891](https://github.com/PyTorchLightning/pytorch-lightning/pull/12891))
```

Reply:
Just realised from your comment and one of your commits https://github.com/PyTorchLightning/pytorch-lightning/pull/12891/commits/c122790c7540e0721f7eeabcde3d2d0ffc9c06e1 that ` torch.ao.quantization.fuse_modules_qat` is only available in PyTorch 1.11+. Sorry for the confusion!